packages/back-end/src/services/features.ts

```diff
@@ -132,6 +132,96 @@ export async function getFeatureDefinitions(
   return defs;
 }
 
+// Exact same as funciton above, but also filters out archived features
+export async function getUnarchivedFeatureDefinitions(
```

Comment:
I don't think we need a separate function for this.  Just add the archive logic to the existing `getFeatureDefinitions` function instead.

Reply:
Changed in recent commit